### ISSUE TITLE
ngf-thumbnail doesn't create src because of error

### ISSUE THREAD

**Issue by Mossar** [danialfarid/ng-file-upload#1076]
Hi, today I decided to use that directive (9.0.12) and it has got many amazing features but I receive error on the easiest part - showing thumbnail. I do it, in my opinion, exactly like in sample.
```html
<input type="file" ngf-select ng-model="data.photo" name="file" accept="image/*" required>
<img ngf-thumbnail="data.photo">
```

I receive following error:
```javascript
Error: b[0].match(...) is null
d.dataUrltoBlob@file:///C:/example/assets/lib/ng-file-upload-all.min.js:2:28850
d.resize/</<@file:///C:/example/assets/lib/ng-file-upload-all.min.js:2:29331
processQueue@file:///C:/example/assets/lib/angular.js:14745:28
scheduleProcessQueue/<@file:///C:/example/assets/lib/angular.js:14761:27
$RootScopeProvider/this.$get</Scope.prototype.$eval@file:///C:/example/assets/lib/angular.js:15989:16
$RootScopeProvider/this.$get</Scope.prototype.$digest@file:///C:/example/assets/lib/angular.js:15800:15
$RootScopeProvider/this.$get</Scope.prototype.$evalAsync/<@file:///C:/example/assets/lib/angular.js:16028:15
completeOutstandingRequest@file:///C:/example/assets/lib/angular.js:5507:7
Browser/self.defer/timeoutId<@file:///C:/example/assets/lib/angular.js:5784:7
```

In result there is in browser just 
```html
<img ngf-thumbnail="data.photo" class="ngf-hide" >
```
The only thing I managed to show it is to remove ngf-hide class and add to img attribute src="{{ bind.$ngfDataUrl }}".

I suppose it should work without those hacks and probably it works, but I do something wrong.

**Comment by danialfarid** [comment]
Thumbnail is usually for the case that you want to show smaller photo than the original one so you need to set the width and height of the img element so the plugin would know what size it should resize it to.
If you wanna show the original selected photo use 'ngf-src="data.photo"`.

I will fix that error in the next version to use the original size if it is not provided.

**Comment by danialfarid** [comment]
Fixed at version 9.0.14

**Comment by Mossar** [comment]
Thank you very much, works well ;)
